src/doc_builder/build_doc.py

```diff
@@ -153,6 +156,7 @@ def build_mdx_files(package, doc_folder, output_dir, page_info):
     output_dir = Path(output_dir)
     os.makedirs(output_dir, exist_ok=True)
     anchor_mapping = {}
+    src_py_path_mapping = {}
```

Comment:
```suggestion
    source_files_mapping = {}
```
Same length but more understandable.

Reply:
handled in https://github.com/huggingface/doc-builder/pull/139/commits/31c4207a7069d765dc2b633d33d698e1e9da986b